### ISSUE TITLE
Show keyboard shortcuts of all Dendron commands (including bundled extensions)

### ISSUE THREAD

**Issue by kevinslin** [dendronhq/dendron#148]

**Comment by kpathakota** [comment]
@kevinslin  - were you thinking something like a shift+? showing a pane of all important commands (like in gmail)? Can't people already see this in the workbench keyboard shortcuts? Maybe we can pre-populate a query to that?

**Comment by kevinslin** [comment]
Yeah, pre populate a query would be great. I'm not sure if vscode supports
that. Other solution is to have a question mark in the preview. But that's
going to be easier once we switch the preview engine
